RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -123,7 +123,6 @@ import Foundation
     /// - Parameter provider: object that will always contain up-to-date user information.
     @objc public static func registerPreference(_ provider: UserInfoProvider) {
         inAppQueue.async {
-            notifyIfModuleNotInitialized()
```

Comment:
Just want to confirm, why is this removed? Is this because `registerPreference` can be set before `configure`?

Reply:
That's right. I think it's better to let clients set `UserInfoProvider` even before init/configure to avoid any issues with API calls order. Only `logEvent` requires `configure` to be called before.